frontend/src/components/managePermissions/table/ApplicationAdminTable.vue

```diff
@@ -73,6 +81,15 @@ const deleteAdmin = (admin: FamAppAdminGetResponse) => {
         },
     });
 };
+
+const highlightNewAppAdminAccesRow = (rowData: any) => {
```

Comment:
I was thinking if we want to make this method reusable for all three tables, but since our new design looks quite different, I feel it's okay to keep it as now. But let's make this style reusable, maybe move the return style into constant, thanks!